docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       - ssh-keys:/root/.ssh
       - ./docker/certificates/ca-cert.pem:/tmp/ca-cert.pem:ro
       - ./docker/certificates/client-cert.pem:/tmp/client-cert.pem:ro
+      - ./docker/certificates/client-cert-and-key.pem:/tmp/client-cert-and-key.pem:ro
```

Comment:
novy subor `client cert + key`, tak to chce MongoDB `connection string` a `CLI`.